lib/generators/sorcery/templates/initializer.rb

```diff
@@ -26,6 +26,11 @@
   #
   # config.cookie_domain =
 
+  # Set name of the remember_me cookie
+  # Default: `remember_me_token`
```

Comment:
This should probably be `:remember_me_token` so it's clear that it's a symbol (and not say, a String) here.